data/hai/hai reveal 6 end.txt

```diff
@@ -50,21 +50,26 @@ mission "Hai Reveal [C01] Cleanup"
 		log `Have learned that during the crisis a single ship loaded with kidnapped Hai was misplaced and abandoned. They require rescue and may indicate the presence of a loose end.`
 		conversation
 			`Arriving on Mountaintop brings with it full diplomatic regalia. The Chancellor of the Free Worlds and President of the Republic are in their most respectable suits, Alondo and Xilin have the traditional symbols of office (though Alondo's looks like the Free Worlds have perhaps not finalized their design just yet), and dress uniforms laden with medals abound on both sides.`
-			`	When you ask around, you find out that Remington and Terry are not invited to such an event, as the Syndicate is technically part of the Republic and awarded no special diplomatic status. "Besides," Xilin mentions, "Terry seems to have disappeared from Syndicate communications, thought I won't hazard a guess as to their internal politics."`
-			`	It seems this station has seen serious renovations in the past weeks. You find yourself standing with the diplomats in a room now called the grand Borealis Chamber, "Where Mountains Touch The Sky," as Teeneep makes the first publicly broadcast speech from the Hai to humanity. The three heads of state announce a resolution granting safe passage to all Hai in human space, as long as they register with customs on Mountaintop before entering, and Teeneep expresses similar sentiments for humans visiting Hai space. It is announced that extensive surveillance fleets will be posted on both sides of the wormhole to ensure safety. This meeting will now turn into a multi-day summit to discuss the creation of a formal treaty. The process will take a number of days.`
+			`	Apparently, since the Syndicate is technically part of the Republic, they are awarded no special diplomatic status and have been busy with corporate matters elsewhere. Xilin finds you between proceedings and warns you, "Terry seems to have disappeared from Syndicate communications, though I won't hazard a guess as to their internal politics." A warning that carries no surprise at this point.`
+			`	It seems this station has seen serious renovations in the past weeks. You find yourself standing with the diplomats in a room now called the grand Borealis Chamber, "Where Mountains Touch The Sky," as Teeneep makes the first publicly broadcast speech from the Hai to humanity. The three heads of state announce a resolution granting safe passage to all Hai in human space, as long as they register with customs on Mountaintop before entering, and Teeneep expresses similar sentiments for humans visiting Hai space. It is announced that extensive surveillance fleets will be posted on both sides of the wormhole to ensure safety. This meeting will now turn into a multi-day summit to discuss the creation of a formal treaty.`
+			`	As you leave the summit for lunch, Xilin and Sayari intercept you. "Trouble. We need to talk," says Xilin.`
```

Comment:
```suggestion
			`	As you leave the summit for lunch, Xilin and Sayari intercept you.`
			`	"Trouble. We need to talk," says Xilin. You follow them to a private meeting room.`
```
Looks better, still could be just a bit more improved. I don't like telling the player what they feel, better for other characters to express that. I'd like to give the player an option to feel frustrated, though, and let Sayari or Xilin talk about "how easy it would be for it to be over".

Reply:
> You follow them to a private meeting room.

I'm trying not to lengthen an already quite long scene by adding another paragraph. It doesn't say where you are when the conversation occurs, so it is reasonable to assume you're in a place where you can discuss such matters. Sayari has been a diplomat longer than a human lifespan, so she'd have the experience to know when they can safely talk.

Reply:
I'd imagine you're leaving the Borealis Chamber the moment Sayari and Xilin meet you again, then they would bring you down to the meeting rooms. Otherwise, the player might feel like they're talking out in the open. Only when you get Sayari to roll her eyes, you go to the meeting rooms.

Reply:
Okay. I put it back in.